dev/MRTCore/build/versions.props

```diff
@@ -24,6 +24,7 @@
     <DotNetCoreRuntimeVersion>5.0.8</DotNetCoreRuntimeVersion>
     <DetoursVersion>4.0.1</DetoursVersion>
     <CppWinRTVersion>2.0.210806.1</CppWinRTVersion>
+    <WILVersion>1.0.210803.1</WILVersion>
```

Comment:
The intent is to standard versions of tooling/dependencies?

Please do this at the PROJECT level. Not just MRT. This should be versions.props in the root and all *proj files should be updated for this standardization. Not just MRT.

We've discussed this a few times but haven't gone here _yet_. Congratulations for being first to the key :-)

There's still the problem of `packages.config`. Given the lack of any #include/import/etc mechanism the best ideas to date include:

* Create a script to crawl the directory tree and update all `packges.config` based on `/versions.props`. Then version updates are (1) edit `/versions.props` and (2) run `/tools/PackagesConfig.cmd -Update` which runs `/tools/PackagesConfig.ps1 -Update` (the usual .ps1/.cmd pairing)
* Same script accepts `-Verify` to scan all `packages.config` and report any version mismatches with `/versions.props`
* Add a task in build pipelines to run `PackagesConfig.cmd -Verify` so builds always report on inconsistencies
* Add `PackagesConfig.*` to the Transport package so other repos can similarly use it. That probably means a 3rd option: `-Update`, `-Verify` and `-Version filename` that defaults to `/versions.props` but lets folks override

We _may_ eventually want to make release builds _enforce_ this consistency but there's always exceptions to rules so better to walk before we run and just _report_ inconsistencies for now.

Thoughts?

Reply:
I must admit that this was somewhat of a hunt-and-peck change. I was adding WILVersion anywhere I saw CppWinRTVersion used/defined. If this file does not affect the C++ project templates then adding the definition here is probably a mistake.

In regards to the issue with `packages.config`, I don't think I know enough about how this project is built to have any meaningful thoughts on it.